.github/workflows/nodejs.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [ 13.x ]
+        node-version: [ 13.x, 14.15.x, 16.1.x ]
```

Comment:
personally, I think it makes sense to only test on node LTS releases (v13 is not LTS). https://nodejs.org/en/about/releases/

Reply:
Yeah, good point. So 12, 14, 16 then it seems.